README.md

```diff
@@ -3,7 +3,14 @@
 
  [ ![Download](https://api.bintray.com/packages/hmrc/releases/help-to-save-stride-frontend/images/download.svg) ](https://bintray.com/hmrc/releases/help-to-save-stride-frontend/_latestVersion)
```

Comment:
can we name the service as `help-to-save-admin-frontend` as `stride` is just authentication mechanism but in this service we are mostly performing admin/customer-service related operations like eligibility check and others...

Reply:
no unit tests written , is this intentional ?

Reply:
i forgot , whoops

Reply:
admin-frontend sounds like something our team would use to control our service in real time - there are other admin-frontends sitting out there which function in this way so it would be confusing. I named it stride-frontend because we're serving up frontend content to users in the stride network. There might be better names, but I don't think admin-frontend is the right one...

Reply:
ok, i cant think of a better name now -  its upto if you keep this name for now